service/templates/admin.html

```diff
@@ -554,6 +555,23 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</md-input-container>
 								</md-card-content>
 							</md-card>
+
+							<md-card flex=50 id="task_limits">
+								<md-card-title>
+									<md-card-title-text>
+										<span>Task Limits</span>
+									</md-card-title-text>
+									<md-button ng-click="clearSection('task_limits')">
+										<i class="fa fa-trash"></i>
+									</md-button>
+								</md-card-title>
+								<md-card-content>
+									<md-input-container class="control" style="width:45%;">
+										<label>Max Tasks per Version</label>
```

Comment:
Do we need a ticket for Spruce changes?

Reply:
there isn't an admin settings page in spruce right now so this is just to update our legacy admin page. I believe there is planned work to make a spruce admin page but no need to ticket for something like this.